internal/datastore/spanner/watch.go

```diff
@@ -189,6 +189,18 @@ func (sd *spannerDatastore) watch(
 				changeRevision := revisions.NewForTime(dcr.CommitTimestamp)
 				modType := dcr.ModType // options are INSERT, UPDATE, DELETE
 
+				// See: https://cloud.google.com/spanner/docs/ttl
+				// > TTL supports auditing its deletions through change streams. Change
+				// > streams data records that track TTL changes to a database have the
+				// > transaction_tag field set to RowDeletionPolicy and the
+				// > is_system_transaction field set to true.
+				if modType == "DELETE" && dcr.TransactionTag == "RowDeletionPolicy" && dcr.IsSystemTransaction {
```

Comment:
isn't there any constant in the spanner SDK? Maybe open an issue in the repo since this seems API that could break any time? Same for `DELETE`

Reply:
Not that I saw, unfortunately